resources/tur.ini

```diff
@@ -1,230 +1,311 @@
 
 ; The /p auto Subcommand
 auto.name=auto
 auto.alias=a
-auto.desc=Rastgele bir arsaya aranıyor
-auto.usage=/p auto [claim: bool] [name: string]
-auto.notplotworld=Arsa dünyasında değilsin
-auto.success="({%0};{%1}) arsası bulundu"
-auto.noplots=Sahipsiz arsa bulunamadı
+auto.desc=§6Bir sonraki boş arsaya ışınlanır
+auto.usage=§e/p auto [claim: bool] [name: string]
+auto.notplotworld=§cSen bir arsa dünyasında değilsin
+auto.success="§aSerbest arsaya ışınlandı §f({%0};{%1})"
+auto.noplots=§cMevcut herhangi bir arsa bulunamadı
 
 ; The /p dispose Subcommand
 dispose.name=dispose
-dispose.desc=Arsanı sahipleri siliyor 
-dispose.usage=/p dispose
-dispose.confirm={%0} arsasını sahipler silmek istediğine emin misin? Eğer onaylıyorsan /p dispose confirm komutunu kullan
-dispose.nomoney=Arsayı sahipler silmek için yeterli paraya sahip değilsin
-dispose.success=Arsa sahipler silindi
+dispose.desc=§6Arsa birakir
+dispose.usage=§e/p dispose
+dispose.confirm=§f{%0} §earsasını elden çıkarmak istediğinizden emin misiniz? Eğer öyleyse, §6/arsa birak onay §eyazınız
+dispose.nomoney=§cBu arsayı elden çıkarmak için yeterli paranız yok
+dispose.success=§aArsa bırakıldı
 
 ; The /p reset Subcommand
 reset.name=reset
-reset.desc=Arsanı siliyor
-reset.usage=/p reset
-reset.confirm={%0} arsasını silmek istediğine emin misin? Eğer onaylıyorsan /p reset confirm komutunu kullan
-reset.nomoney=Arsayı sıfırlamak için yeterli paraya sahip değilsin
-reset.success=Arsa sıfırlandı
+reset.desc=§6Arsayı bırakır ve sıfırlar
+reset.usage=§e/p reset
+reset.confirm=§f{%0} §earsasını sıfırlamak istediğinizden emin misiniz? Eğer öyleyse, §6/arsa sifirla onay §eyazınız
+reset.nomoney=§cBu arsayı sıfırlamak için yeterli paranız yok
+reset.success=§aArsa sıfırlandı
 
 ; The /p clear Subcommand
 clear.name=clear
-clear.desc=Arsanı Boşaltıyor
-clear.usage=/p clear
-clear.confirm={%0} arsasını silmek istediğine emin misin? Eğer onaylıyorsan /p clear confirm komutunu kullan
-clear.nomoney=Arsayı temizlemek için yeterli paraya sahip değilsin
-clear.success=Arsa temizlendi
+clear.desc=§6Arsanı temizler
+clear.usage=§e/p clear
+clear.confirm=§f{%0} §egrafiğini temizlemek istediğinizden emin misiniz? Eğer öyleyse, §6/arsa temizle onaylayın §eyazınız 
+clear.nomoney=§cBu arsayı temizlemek için yeterli paranız yok
+clear.success=§aArsa temizlendi
 
 ; The /p biome Subcommand
 biome.name=biome
-biome.desc=Arsanın biomunu değiştir
-biome.usage=/p biome [biome: string]
-biome.success=Arsa biomu {%0} olarak ayarlandı
-biome.invalid=Seçtiğiniz biom bulunamadı
-biome.possible=Olası biyomlar: {%0}
+biome.desc=§6Arsa biyomunu değiştirirsin
+biome.usage=§e/p biome [biome: string]
+biome.success=§aArsa biyomu §f{%0} §aolarak değiştirildi
+biome.invalid=§cBu biyom yok
+biome.possible=§eOlası Biyomlar: §f{%0}
+biome.form=Biyom
 
 ; The /p biomes Subcommand
-biomes.name=biomes
-biomes.desc=Biom listesini görüntüler
-biomes.usage=/p biomes
-biomes.possible=Olası biomlar: {%0}
+biomes.name=biome
```

Comment:
Please leave this as `biomes`. This is separate from the biome command listed above.

Reply:
ok sorry

Reply:
> Please leave this as `biomes`. This is separate from the biome command listed above.

i fixed it.